things/service.go

```diff
@@ -180,6 +199,9 @@ func (ts *thingsService) CreateThings(ctx context.Context, token string, things
 				return []Thing{}, errors.Wrap(ErrCreateUUID, err)
 			}
 		}
+		if err := ts.addPolicy(ctx, res.GetId(), things[i].ID, ownerRelationKey); err != nil {
```

Comment:
we can put logic for creation of a thing and adding a policy in additionl function that would be called from within a loop
also, what happens with a created thing when adding policy has failed?

Reply:
When adding policy fails, the service returns an error; although, the things are saved into the DB. It should be handled as well.